### ISSUE TITLE
try-prisma: command not found

### ISSUE THREAD

**Issue by JulianNicholls** [prisma/try-prisma#68]
When I try `npx try-prisma` on an M1 MacBook, all I get is `sh: try-prisma: command not found`.
This is with version 1.1.1 (latest)

Using try-prisma@1.0.58, it works.

**Comment by ViniciusCestarii** [comment]
Same happened for me.

Windows 10

npm version: 10.2.3

Using try-prisma@1.0.58 worked for me too

**Comment by mattmorganpdx** [comment]
same. I'm on linux
```
$ npm --version
10.9.0
$ cat /etc/lsb-release 
DISTRIB_ID=Ubuntu
DISTRIB_RELEASE=24.04
DISTRIB_CODENAME=noble
DISTRIB_DESCRIPTION="Ubuntu 24.04.1 LTS"
```
@ViniciusCestarii advice worked using 1.0.58 and then I got through the rest of setup steps okay.

Also I just want to say I'm very excited about the work y'all are doing. cheers

**Comment by JulianNicholls** [comment]
This is now resolved in version 1.2.0
